### PR TITLE
profile add profileTime

### PR DESCRIPTION
**Pull request by even986025158**
## What type of PR is this：
- [ ] enhancement

profile add profileTime

## Problem Summary(Required) ：
the profile does not have profileTime; add it to profile

### CODE REVIEW

**Comment by mofeiatwork** [comment]
Did you encounter any performance issue of profile? Why add this property in profile?

**Comment by chaoyli** [comment]
@even986025158 Need you to sign the CLA

**Comment by even986025158** [comment]
> @even986025158 Thanks for your contributions. Need you to sign the CLA.

when I click into [Contributor License Agreement](https://cla-assistant.io/StarRocks/starrocks?pullRequest=6328)] it shows than You have agreed to the CLA for StarRocks/starrocks

**Comment by even986025158** [comment]
> Did you encounter any performance issue of profile? Why add this property in profile?

Because sometimes after the profile is turned on, some execution time will take longer than normal, so I need to see how much profileTime costs

File: fe/fe-core/src/main/java/com/starrocks/common/util/ProfileManager.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ public class ProfileManager {
     public static final String SQL_STATEMENT = "Sql Statement";
     public static final String USER = "User";
     public static final String DEFAULT_DB = "Default Db";
+    public static final String PROFILE_TIME = "Profile Time";
```
Comment (mofeiatwork) [inline fe/fe-core/src/main/java/com/starrocks/common/util/ProfileManager.java:64]: Cost or Duration?

**Comment by mofeiatwork** [comment]
run all

**Comment by imay** [comment]
@even986025158 You need use the same email address in the commit as the one you login in GitHub. Please check it. Other the CLA can't pass

File: fe/fe-core/src/main/java/com/starrocks/common/util/ProfileManager.java (hunk 1 above)
Comment (even986025158) [reply fe/fe-core/src/main/java/com/starrocks/common/util/ProfileManager.java:64]: Duration

**Comment by even986025158** [comment]
> @even986025158 You need use the same email address in the commit as the one you login in GitHub. Please check it. Other the CLA can't pass

I have  add the other email in GitHub, can it ok now

**Comment by imay** [comment]
@even986025158 You should change you git config to your github email and push it again. I think you can close this PR and create an new PR with the new commit.

**Pull request closed without merging**